core/domain/activity_jobs_one_off.py

```diff
@@ -72,6 +76,36 @@ def reduce(key, values):
             yield (key, values)
 
 
+class ActivityContributorsSummaryOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Do you need to add this to the job registry?

Reply:
Yes, thanks.